src/main/java/org/folio/rest/helper/ExchangeHelper.java

```diff
@@ -35,9 +35,9 @@ public ExchangeRate getExchangeRate(String from, String to) {
     }
   }
 
-  public Double calculateExchange(String sourceCurrency, String targetCurrency, Number amount) {
+  public Double calculateExchange(String sourceCurrency, String targetCurrency, Number amount, Number customRate) {
     var initialAmount = Money.of(amount, sourceCurrency);
-    var rate = getExchangeRate(sourceCurrency, targetCurrency).getExchangeRate();
+    var rate = customRate == null ? getExchangeRate(sourceCurrency, targetCurrency).getExchangeRate() : customRate;
```

Comment:
I suggest to use ObjectUtils.defaultIfNull() method

Reply:
I personally like to use just equal operation instead of object method for being easy to read, and it is Number so it is okay to remain

Reply:
@azizbekxm I don't think it's appropriate to merge a PR without resolving all the comments.

Reply:
Oh, sorry. https://issues.folio.org/browse/UINV-522 ticket is taking by Alisher and it is progress, not to block him I thought it is better to merge

Reply:
please keep the reviewer informed on future occasions. thanks